x/lsnative/staking/types/keys.go

```diff
@@ -54,6 +57,7 @@ var (
 	TokenizeShareRecordIdByOwnerPrefix = []byte{0x62} // key for tokenizeshare record id by owner prefix
 	TokenizeShareRecordIdByDenomPrefix = []byte{0x63} // key for tokenizeshare record id by denom prefix
 	LastTokenizeShareRecordIdKey       = []byte{0x64} // key for last tokenize share record id
+	TotalLiquidStakedTokensKey         = []byte{0x65} // key for total liquid staked tokens
```

Comment:
Should we add migrations for these keys?

Reply:
Added